package.json

```diff
@@ -97,7 +100,7 @@
       [
         "@semantic-release/exec",
         {
-          "prepareCmd": "npx run-s generate-docs"
+          "prepareCmd": "npx run-s docs"
```

Comment:
Not sure why the `exec` script in run this way through `npx run-s` vs. using `yarn` (are dependencies not installed before it? Is `yarn` not available? Are paths solved differently?). In any case, I left it as is fearing it'd break CI if I changed it.

Reply:
Same applies to [`ramda-land`, BTW](https://github.com/flybondi/ramda-land/pull/20/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R130).